tests/sandbox/SandboxTest.py

```diff
@@ -130,6 +130,21 @@ def call_put_endpoint_url_with_request(
             headers=headers,
         )
 
+    @pytest.fixture
+    def call_endpoint_url_with_value_and_request(
+        self,
+        send_rest_request: Callable[[HttpMethod, str, Actor], Response],
+        load_json: Callable[[str], Dict[str, str]],
+        endpoint_url: str,
+    ) -> Callable[[Actor, str, str], Response]:
+        return lambda actor, requestJson, param, headers={}: send_rest_request(
```

Comment:
It might be a bit more flexible for `param` to instead be a dictionary mapping template values to their respective replacements. I think this could then be supplied to the `endpoint_url.format` method.